projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -211,6 +215,60 @@ public static function should_initialize() {
 		return $should;
 	}
 
+	/**
+	 * Extend the actions links with My Jetpack
+	 * for all Jetpack plugins.
+	 *
+	 * @return void
+	 */
+	public static function extend_jetpack_plugins_action_links() {
+		// Plugins list to add to the My Jetpack action link.
+		$jetpack_plugins = array_unique(
+			array(
+				Products\Backup::get_plugin_slug(),
+				Products\Boost::get_plugin_slug(),
+				Products\Crm::get_plugin_slug(),
+				Products\Search::get_plugin_slug(),
+				'jetpack',
+			)
+		);
+
+		$all_plugins = Plugins_Installer::get_plugins();
```

Comment:
Do we really need to get all this list in loop through it every time? Wouldn't it work if we simply added the hooks to the plugin filenames we already have in the Products classes?

Reply:
yes, that's a better approach I think. 🤔